### PR TITLE
Increased test-coverage around Sources

### PR DESCRIPTION
**Pull request by eminn**
Increased coverage of `Sources.class` to 100%

### CODE REVIEW

**Comment by eminn** [comment]
verify

File: hazelcast-jet-core/src/test/java/com/hazelcast/jet/SourcesTest.java [hunk 1]
```diff
@@ -16,20 +16,60 @@
...
 import static java.util.stream.Collectors.toList;
 import static org.junit.Assert.assertEquals;
 
 public class SourcesTest extends PipelineTestSupport {
+    private static HazelcastInstance hz;
+    private static ClientConfig clientConfig;
+
+    @BeforeClass
+    public static void setUp() throws Exception {
+        hz = Hazelcast.newHazelcastInstance();
```
Comment (cangencer) [inline hazelcast-jet-core/src/test/java/com/hazelcast/jet/SourcesTest.java]: can we rename this to `remoteHz`? They can also be moved to the base class as `createRemoteCluster` as I see this code duplicated.

File: hazelcast-jet-core/src/test/java/com/hazelcast/jet/SourcesTest.java [hunk 2]
```diff
@@ -16,20 +16,60 @@
...
+        Hazelcast.newHazelcastInstance();
+
+        clientConfig = new ClientConfig();
+        clientConfig.getGroupConfig().setName("dev");
+        clientConfig.getGroupConfig().setPassword("dev-pass");
+        Address address = hz.getCluster().getLocalMember().getAddress();
+        clientConfig.getNetworkConfig().addAddress(address.getHost() + ':' + address.getPort());
```
Comment (cangencer) [inline hazelcast-jet-core/src/test/java/com/hazelcast/jet/SourcesTest.java]: are these required? these look like the default values?

File: hazelcast-jet-core/src/test/java/com/hazelcast/jet/SourcesTest.java [hunk 3]
```diff
@@ -89,85 +129,214 @@ public void mapWithFilterAndProjectionFn() {
...
+                .drainTo(sink);
+        execute();
+
+        // Then
+        int nodeCount = jet().getCluster().getMembers().size();
+        assertEquals(4 * nodeCount, sinkList.size());
+    }
+
+    @Test
+    @Ignore
```
Comment (cangencer) [inline hazelcast-jet-core/src/test/java/com/hazelcast/jet/SourcesTest.java]: needs ignore reason

File: hazelcast-jet-core/src/test/java/com/hazelcast/jet/Sources_withEventJournalTest.java [hunk 4]
```diff
@@ -34,19 +44,56 @@
...
+    private static HazelcastInstance hz;
+    private static ClientConfig clientConfig;
+
+    @BeforeClass
+    public static void setUp() throws Exception {
+        Config config = new Config();
+        config.getMapEventJournalConfig(JOURNALED_MAP_PREFIX + '*').setEnabled(true);
+        config.getCacheEventJournalConfig(JOURNALED_CACHE_PREFIX + '*').setEnabled(true);
+        hz = Hazelcast.newHazelcastInstance(config);
+        Hazelcast.newHazelcastInstance(config);
```
Comment (cangencer) [inline hazelcast-jet-core/src/test/java/com/hazelcast/jet/Sources_withEventJournalTest.java]: see my comments about remote hz cluster above

**Pull request merged**